x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/container/QueryContainer.java

```diff
@@ -227,7 +230,7 @@ public int limit() {
 
     public boolean isAggsOnly() {
```

Comment:
Should we rename the method since with `anyMatch` the name seems misleading now?

Reply:
The name is a bit misleading but the meaning holds, the method indicates whether the query is an agg-only or scroll-any.
The issue with some expressions is that they can appear in both contexts (column name or literals), hence the `anyMatch`.
This might be addressed better through `FieldExtraction` directly.

Reply:
Maybe `hasAggsOnly` is better?